src/components/colorPalette/ColorPalette.api.json

```diff
@@ -31,7 +31,7 @@
     },
     {
       "name": "onValueChange",
-      "type": "(value: string, options: object) => void",
+      "type": "(color: string, extraData: ColorExtraData) => void",
```

Comment:
Since the props is named 'onValueChange' I'd leave it as 'value'. Also since the user probably already extracting 'value' here